### PR TITLE
Added default value to useComputedState in the CKEDITOR config object

### LINKED ISSUE #4918: Default value for useComputedState is undefined but treated as `true` value.
## Type of report

Bug

## Provide detailed reproduction steps (if any)

Documentation says that default value for [useComputedState](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-useComputedState) is `true`. However, there is no place in the code that assigns `true` to this config option.

This option is used in `justify` plugin, which interprets the `undefined` value as `true`: https://github.com/ckeditor/ckeditor4/blob/728e1923987dbf1ad9804a35e4b7945788025535/plugins/justify/plugin.js#L12

This option is used in `bidi` plugin, which interprets the `undefined` value as `true`: 
https://github.com/ckeditor/ckeditor4/blob/728e1923987dbf1ad9804a35e4b7945788025535/plugins/bidi/plugin.js#L18

This option is used in `list` plugin, which interprets the `undefined` value as `true`: 
https://github.com/ckeditor/ckeditor4/blob/728e1923987dbf1ad9804a35e4b7945788025535/plugins/list/plugin.js#L367

Please note, that there are few other lines in those plugins, that use this 'trick'.

Code logic makes the value of `config.useComputedState` interpreted the same as the documentation says: as `true` value. However, If the user tries to use `config.useComputedState`, he will get `undefined` instead of `true`.

### Expected result

*What is the expected result of the above steps?*
Plain `config.useComputedState` config option is equal to `true`.

### Actual result

*What is the actual result of the above steps?*

Plain `config.useComputedState` config option is equal to `undefined` and interpreted as `true`.

## Other details

* Browser: -
* OS: -
* CKEditor version: 4.16.2 (it works like that since the option was introduced)
* Installed CKEditor plugins: -

### PR DESCRIPTION
**Pull request by shabab477**
Issue #4918

## Does your PR contain necessary tests?

All patches that change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [x] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?
Skipped

## What changes did you make?

Added default config value for `useComputedState`

## Which issues does your PR resolve?

Closes #4918.

### CODE REVIEW

**Review by Comandeer — CHANGES_REQUESTED** [review]
Hi, @shabab477! Thanks for your contribution.

The change looks good. However, we require that every change be accompanied by the test. Could I ask you to add a test to `/tests/config` directory? The test can be simple, e.g. creating the editor and then checking its `editor.config.useComputedState` value.

It'd be also good to remove the logic connected to treating undefined `config.useComputedState` variable the same way as the one set to `true` (see the issue's description for more info). Could you add it as a part of this PR?

**Comment by shabab477** [comment]
Alright :+1:

File: tests/core/config/inline.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ bender.test( {
 		} );
 	},
 
+	test_startup_computed_state_value: function() {
+		assert.isTrue( this.editor.config.useComputedState === true, 'config.useComputedState should return true as default' );
```
Comment (Comandeer) [inline tests/core/config/inline.js:26]: There is no need to compare variable's value to `true`, asserting its value is enough in this case:

```suggestion
		assert.isTrue( this.editor.config.useComputedState, 'config.useComputedState should return true as default' );
```

**Review by Comandeer — CHANGES_REQUESTED** [review]
Thanks for your fixes!

The test looks good. Could I ask you to add a similar one also to `tests/core/config/themed_ui.js`?

There are also several other places in plugins where the trick similar to the one with `undefined` is used, e.g. https://github.com/ckeditor/ckeditor4/blob/6111ac2d0896e1bc53060ea1404c63612a37b5b1/plugins/bidi/plugin.js#L83. Could I ask you to also fix them?

File: core/config.js [hunk 2]
```diff
@@ -369,6 +369,22 @@ CKEDITOR.config = {
...
+	 *
+	 * If enabled in a Left-To-Right written document, the "Left Justify"
+	 * alignment button will be shown as active, even if the alignment style is not
+	 * explicitly applied to the current paragraph in the editor.
+	 *
+	 *		config.useComputedState = false;
+	*
+	* @since 3.4.0
+	* @cfg {Boolean} [useComputedState=true]
+	*/
```
[the remark this answers is not in the document]
Comment (shabab477) [reply core/config.js]: Done

File: tests/core/config/inline.js (hunk 1 above)
Comment (shabab477) [reply tests/core/config/inline.js]: Done

**Comment by shabab477** [comment]
Fixed usages of  `useComputedState` workarounds in the code for plugins:
- justify
- list
- bidi

**Review by Comandeer — APPROVED** [review]
LGTM! Thanks once again for your contribution.

**Pull request merged**